testing/workflows/components/workflows.libsonnet

```diff
@@ -652,14 +633,6 @@
               "copy_artifacts",
               "--bucket=" + bucket,
             ]),  // copy-artifacts
-            buildTemplate("jsonnet-test", [
-              "python",
-              "-m",
-              "testing.test_jsonnet",
-              "--artifacts_dir=" + artifactsDir,
-              "--test_files_dirs=" + srcDir + "/kubeflow",
-              "--jsonnet_path_dirs=" + srcDir,
```

Comment:
Do we need --jsonnet_path_dirs=" + srcDir + "," + srcRootDir + "/kubeflow/testing/workflows/lib/v1.7.0"

Reply:
Not in this file. This Argo workflow should not be running the jsonnet tests.

We use unit_tests.jsonnet workflow to run the jsonnet unittests and you already updated that file to set jsonnet_path_dirs
https://github.com/kubeflow/kubeflow/blob/master/testing/workflows/components/unit_tests.jsonnet#L116